JavaScript/io.js

```diff
@@ -0,0 +1,75 @@
+const prompt = require("prompt-sync")({ sigint: true });
+const Score = require("./score");
+
+class Io {
+  constructor() {
+    this.frames = [];
+  }
+
+  runGame() {
+    console.log("Welcome to 10 pin bowling!");
+    console.log("Enter the number of pins knocked down");
+    this.enterFrames();
+    const score = new Score(this.frames);
+    console.log(`Congratulations you scored: ${score.total()}`);
+  }
+
+  enterFrames() {
+    for (let f = 0; f < 10; f++) {
+      console.log(`Frame ${f + 1}`);
+      console.log("---------------");
+      console.log("frame: " + this.frames[f]);
+      this.promptRolls();
+      if (
+        f === 9 &&
+        (this.frames[f][0] + this.frames[f][1] === 10 ||
+          this.frames[f][1] === 10)
```

Comment:
As discussed, this will benefit from using the isStrike and isSpare function in the Score class so that if someone reads your code, it is a bit clearer.